FAQ.md

```diff
@@ -0,0 +1,151 @@
+
+In the "daily update", the stream with the best supported resolution will be considered "priority" over other lower resolution streams of the same channel.
+
+Based on [#1916](https://github.com/iptv-org/iptv/issues/1916).
+
+### Why don't we accept Xtream-Codes servers inside our playlists?
+Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it or the server that hosts it.
+
+In short, they are poor quality streams created by unauthorized people.
+
+If you're unsure about if your link is from an Xtream-Codes server, you may:
+* Look at the structure of the URL. Most of them have this form : http(s)://*hostname*:25461/*username*/*password*/*channelID* (port is often 25461)
+* Remove the rest of the URL after the password mention and add either "panel_api.php?" or "player_api.php?" after the port. Replace the slashes between the username and password with "username=" and "password=". 
+
+If the link matches or answers after changing the URL, you're with an Xtream-Codes server.
+
+Based on [#5401](https://github.com/iptv-org/iptv/pull/5401).
+
+### Is it possible to add streams from Youtube, Dailymotion or Twitch?
+Yes. Due to technical limitations, it is necessary to add a [Streamlink](https://streamlink.github.io/) server link to view the content.
+
+### Is it possible to add audio-only streams, like FM radio?
+No. Exceptions are the visual radios, in which a video and audio are shown at the same time. A WIP repository made by one member of iptv-org, [LaneSh4d0w](https://github.com/LaneSh4d0w) aims to collect radio streams, on the [IPRD](https://github.com/LaneSh4d0w/IPRD) repository.
+
+### Why are there some call sign (KJLA-DT1.us) as ids and alphanumeric names as ids (Mychannel.us)?
+It's to differentiate the origin of the broadcast from the content. Call signs usually come from physical stations in the country of origin and their programming is subject to change. If the channel is not from a television station, the brand name will be used.
+
+An example of a call sign used as id is `KJLA-DT1.us`: [KJLA](https://en.wikipedia.org/wiki/KJLA) is a digital television station, DT is a suffix, 1 is a subchannel number and ".us" is the country code (United States). This station broadcasts the Visión Latina channel, whose id is `VisionLatina.us`.
+
+### There are source for verify call sign and coverage area for some TV stations?
+There are websites to consult the location and characteristics of the station. For example, [FCCData](https://fccdata.org/) for US, Canada, Mexico, UK, Australia and Japan.
+
```

Comment:
```suggestion
```

Reply:
I added several page about these questions. The last question is understandable because it is for those who need to find a source for television stations.